### ISSUE TITLE
Breaks when actually using libc

### ISSUE THREAD

**Issue by DavidPoliakoff** [LLNL/GOTCHA#2]
@mplegendre ,

With the new libc_wrappers.h

```

[  2%] Building C object src/CMakeFiles/gotcha.dir/gotcha_utils.c.o
In file included from /g/g0/dzpolia/src/GOTCHA/src/gotcha_utils.c:17:
/g/g0/dzpolia/src/GOTCHA/src/libc_wrappers.h:40:35: error: "(" may not appear in macro parameter list
make[2]: *** [src/CMakeFiles/gotcha.dir/gotcha_utils.c.o] Error 1
make[1]: *** [src/CMakeFiles/gotcha.dir/all] Error 2
make: *** [all] Error 2
@dzpolia:vim ../src/libc_wrappers.h
@dzpolia:git checkout -- ../src/libc_wrappers.h
@dzpolia:cmake -DGOTCHA_ENABLE_TESTS=ON -DCMAKE_C_FLAGS="-DGOTCHA_USE_LIBC" ..
-- Configuring done
-- Generating done
-- Build files have been written to: GOTCHA/build
@dzpolia:make
Scanning dependencies of target gotcha
[  2%] Building C object src/CMakeFiles/gotcha.dir/gotcha_utils.c.o
[  5%] Building C object src/CMakeFiles/gotcha.dir/gotcha.c.o
[  7%] Building C object src/CMakeFiles/gotcha.dir/gotcha_auxv.c.o
GOTCHA/src/gotcha_auxv.c: In function 'parse_auxv_contents':
GOTCHA/src/gotcha_auxv.c:48: error: expected expression before ')' token
GOTCHA/src/gotcha_auxv.c: In function 'get_vdso_from_maps':
GOTCHA/src/gotcha_auxv.c:228: error: expected expression before ')' token
make[2]: *** [src/CMakeFiles/gotcha.dir/gotcha_auxv.c.o] Error 1
make[1]: *** [src/CMakeFiles/gotcha.dir/all] Error 2
make: *** [all] Error 2
```

I'll start knocking that out tomorrow

**Comment by DavidPoliakoff** [comment]
References #1

**Comment by mplegendre** [comment]
Fixed by PR #3
